src/main/java/com/hlag/rulemaker/RuleMakerJsonLogic.java

```diff
@@ -0,0 +1,41 @@
+package com.hlag.rulemaker;
+
+import com.hlag.rulemaker.expression.ClampExpression;
+import com.hlag.rulemaker.expression.CustomMathExpression;
+import com.hlag.rulemaker.expression.CustomNumericComparisonExpression;
+import com.hlag.rulemaker.expression.DateDiffExpression;
+import io.github.jamsesso.jsonlogic.JsonLogic;
+import io.github.jamsesso.jsonlogic.JsonLogicException;
+
+public class RuleMakerJsonLogic {
+
+    private final JsonLogic jsonLogic;
+
+    public RuleMakerJsonLogic() {
```

Comment:
at least tests that check that all custom operation are added should be added.

Reply:
tested in `RuleMakerUnitTest`